src/common/constants.ts

```diff
@@ -52,9 +53,27 @@ export class Constants {
     public static CSHARP_FUNCTION_DESCRIPTION = "Create an Azure Function and deploy to IoT Edge";
     public static MACHINE_LEARNING_DESCRIPTION = "Deploy Azure Machine Learning images to Azure IoT Edge";
     public static STREAM_ANALYTICS_DESCRIPTION = "Deploy Azure Stream Analytics to Azure IoT Edge";
+    public static EVENT_GRID_DESCRIPTION = "Deploy Azure Event Grid to Azure IoT Edge";
     public static ACR_MODULE_DESCRIPTION = "Import an existing module image from your Azure Container Registry";
     public static EXISTING_MODULE_DESCRIPTION = "Import an existing module image from any container registry";
     public static MARKETPLACE_MODULE_DESCRIPTION = "Import an existing module image from Azure Marketplace";
+    public static EVENT_GRID_IMAGE = "mcr.microsoft.com/azure-event-grid/iotedge:latest";
+    public static EVENT_GRID_CREATE_OPTIONS = {
+        Env: [
+            "inbound:clientAuth:clientCert:enabled=false",
+            "inbound:serverAuth:tlsPolicy=enabled",

```

Comment:
is this line added in marketplace module? https://docs.microsoft.com/zh-cn/azure/event-grid/edge/pub-sub-events-webhook-local didn't contain this line

Reply:
I refer to the marketplace one: https://gallery.azure.com/artifact/20161101/azure-event-grid.event-grid-edgepublic-preview.1.0.0/Artifacts/iot-edge-metadata.json
It would be more accurate